metricbeat/docs/modules/aws.asciidoc

```diff
@@ -10,10 +10,12 @@ This file is generated! See scripts/mage/docs_collector.go
 
 This module periodically fetches monitoring metrics from AWS CloudWatch using
 https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricData.html[GetMetricData API] for AWS services.
-Note: extra AWS charges on GetMetricData API requests will be generated by this module.
 
 All metrics are enabled by default.
 
+IMPORTANT: Extra AWS charges on CloudWatch API requests will be generated by this
```

Comment:
```suggestion
IMPORTANT: Extra AWS charges on CloudWatch API requests may be generated by this
```

Reply:
This one is a `will` because there is no free API calls.